testing/load-test/v2/produce-binary-to-partition-test.js

```diff
@@ -0,0 +1,58 @@
+import {sleep} from "k6";
+import encoding from "k6/encoding";
+
+import {randomIntBetween, randomItem, uuidv4} from "https://jslib.k6.io/k6-utils/1.0.0/index.js";
+
+import {produceBinaryToPartition} from "./common.js";
+import {createTopic, deleteTopic, listClusters, listPartitions} from "../v3/common.js";
+
+export let options = {
+    stages: [
+        {duration: '10s', target: 100},
+        {duration: '40s', target: 100},
+        {duration: '10s', target: 0},
+    ],
+    setupTimeout: '1m',
+    teardownTimeout: '1m'
+};
+
+export function setup() {
+    let listClustersResponse = listClusters();
+    let clusterId = randomItem(listClustersResponse.json().data).cluster_id;
+
+    let topics = [];
+    let partitions = [];
+
+    for (let i = 0; i < 10; i++) {
+        let topicName = `topic-${uuidv4()}`;
+        topics.push(topicName);
+
+        createTopic(clusterId, topicName, 10, 3);
+        sleep(1);
+
+        let listPartitionsResponse = listPartitions(clusterId, topicName);
+        listPartitionsResponse.json().data.forEach(
+            partition => partitions.push({topicName, partitionId: partition.partition_id}));
+    }
+
+    return {clusterId, topics, partitions};
+}
+
+export default function (data) {
+    let records = [];
+    let size = randomIntBetween(1, 100);
```

Comment:
what is the goal of these tests? if performance regression detection is the goal, I guess using a distribution of batch sizes detects broader based changes in performance across the parameter space, and if you had to choose one measure for this specific goal, this is probably what you want. an alternative would be to choose a batch size that optimizes for throughput/latency. you could also argue that's important because it's a better number to brag about. a change in this would also be a pretty good indicator of change across other parts of the parameter space i think.

(no changes requests, just giving some commentary).